cpu/cc3200/include/cc3200_timer.h

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright (C) 2019 Ludwig Maximilian Universität
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup        cpu_cc3200_timer CC3200 General Purpose Timer
+ * @ingroup         cpu_cc3200_regs
+ * @{
+ *
+ * @file
+ * @brief           Driver for the cc3200 timer controller
+ *
+ * Header file with register and macro declarations for the cc3200 Timer module
+ *
+ * @author          Wladislaw Meixner <wladislaw.meixner@campus.lmu.de>
+ *
+ * @{
+ */
+
+#ifndef CC3200_TIMER_H
+#define CC3200_TIMER_H
+
+#include "cc3200.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Timer interrupts
+ * @{
+ */
+#define TIMER_TIMB_DMA 0x00002000     /**< TimerB DMA Done interrupt */
+#define TIMER_TIMB_MATCH 0x00000800   /**< TimerB match interrupt */
+#define TIMER_CAPB_EVENT 0x00000400   /**< CaptureB event interrupt */
+#define TIMER_CAPB_MATCH 0x00000200   /**< CaptureB match interrupt */
+#define TIMER_TIMB_TIMEOUT 0x00000100 /**< TimerB time out interrupt */
+#define TIMER_TIMA_DMA 0x00000020     /**< TimerA DMA Done interrupt */
+#define TIMER_TIMA_MATCH 0x00000010   /**< TimerA match interrupt */
+#define TIMER_CAPA_EVENT 0x00000004   /**< CaptureA event interrupt */
+#define TIMER_CAPA_MATCH 0x00000002   /**< CaptureA match interrupt */
+#define TIMER_TIMA_TIMEOUT 0x00000001 /**< TimerA time out interrupt */
+/** @} */
+
+/**
+ * @brief Timer/Subtimer selection
+ * @{
+ */
+#define TIMER_A 0x000000ff    /**< Timer A */
+#define TIMER_B 0x0000ff00    /**< Timer B */
+#define TIMER_BOTH 0x0000ffff /**< Timer Both */
+/** @} */
+
+/**
+ * @brief Timer configurations
+ * @{
+ */
+#define TIMER_CFG_A_ONE_SHOT 0x00000021 /**< Timer A one-shot timer */
+#define TIMER_CFG_A_ONE_SHOT_UP \
+    0x00000031                          /**< Timer A one-shot up-count timer */
+#define TIMER_CFG_A_PERIODIC 0x00000022 /**< Timer A periodic timer */
+#define TIMER_CFG_A_PERIODIC_UP \
+    0x00000032                           /**< Timer A periodic up-count timer */
```

Comment:
Same here - put the comment above the define if it doesn't fit on the line, so you can keep the define on a single line.

Reply:
should I be using `@brief` when commenting above in that way?

Reply:
Uh I hope not. If this is relevant for Doxygen, I'd say rather ignore the 80 character limit than splitting the define in two.